cmd/plugin.go

```diff
@@ -47,6 +47,25 @@ type pluginJsonOutput struct {
 	Warnings  []string          `json:"warnings"`
 }
 
+type configFileOption struct {
```

Comment:
```suggestion
type pluginInstallConfig struct {
```
Move these into ociinstaller package